benchexec/tools/jcwit.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+from benchexec.tools.template import ToolNotFoundException
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+
+    """
+    Tool info for jcwit
+    (https://github.com/Chriszai/JCWIT).
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("jcwit.py")
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def name(self):
+        return "jcwit"
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable] + options + list(task.input_files)
+
+    def determine_result(self, run):
+        output = run.output
+        validation = "unknown"
+        for line in output:
+            if "Exception" in line:
+                if "AssertionError" in line:
+                    validation = "false"
+                else:
+                    validation = "unknown"

```

Comment:
If the tool crashes, this should typically not be hidden in result `RESULT_UNKNOWN` but reflected with `RESULT_ERROR`. `RESULT_UNKNOWN` is for cases where the tool voluntarily gives up (e.g., because of a conceptually incomplete method).

It also seems weird that any kind of `AssertionError` produces `RESULT_FALSE_PROP`. Certainly a tool crash with `AssertionError` is not (always) indicating that the validation result is `FALSE`?